charts.d/apcupsd.chart.sh

```diff
@@ -43,11 +44,12 @@ apcupsd_check() {
     for host in "${!apcupsd_sources[@]}"
     do
         run apcupsd_get "${apcupsd_sources[${host}]}" >/dev/null
+        # shellcheck disable=2181
         if [ $? -ne 0 ]
         then
             error "cannot get information for apcupsd server ${host} on ${apcupsd_sources[${host}]}."
             failed=$((failed + 1))
-        elif [ $(apcupsd_get "${apcupsd_sources[${host}]}" | awk '/^STATUS.*/{ print $3 }') != "ONLINE" ]
+        elif [ "$(apcupsd_get \"${apcupsd_sources[${host}]}\" | awk '/^STATUS.*/{ print $3 }')" != "ONLINE" ]
```

Comment:
are you sure backslases are needed here? They should not, since they are inside `$()`.